api/envoy/service/load_stats/v2alpha/orca.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package envoy.service.load_stats.v2alpha;
+
+option java_outer_classname = "OpenRCAProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.service.load_stats.v2alpha";
+option go_package = "v2alpha";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// Out-of-band (OOB) load reporting service for frontends to periodically get
```

Comment:
We probably need a bunch more comments here. Let's discuss tomorrow on IM, but I think the first step is for me to create a public version of the design doc and share this as an Envoy issue, then we can crib from that.